src/resources/TemporaryTokenAlias.ts

```diff
@@ -77,28 +77,36 @@ export class TemporaryTokenAlias extends CRUDResource {
 
     create(
         data: SendData<TemporaryTokenAliasCreateParams>,
+        auth?: AuthParams,
         callback?: ResponseCallback<ResponseTemporaryTokenAlias>
     ): Promise<ResponseTemporaryTokenAlias> {
-        return this.defineRoute(HTTPMethod.POST, "/tokens/alias", TemporaryTokenAlias.requiredParams)(data, callback);
+        return this.defineRoute(HTTPMethod.POST, "/tokens/alias", TemporaryTokenAlias.requiredParams)(
+            data,
+            callback,
+            auth
+        );
     }
 
     get(
         storeId: string,
         id: string,
         data?: SendData<TemporaryTokenAliasQrOptions>,
+        auth?: AuthParams,
         callback?: ResponseCallback<Blob>
     ): Promise<Blob>;
     get(
         storeId: string,
         id: string,
         data?: SendData<TemporaryTokenAliasParams>,
+        auth?: AuthParams,
         callback?: ResponseCallback<ResponseTemporaryTokenAlias>
     ): Promise<ResponseTemporaryTokenAlias>;
```

Comment:
Not for this PR. Just curiosity. Is it for typing? 🤔 Looks like a pretty weird structure as the `get` function is implemented after.

Reply:
Wondered about this one too while refactoring. Honestly don't know, is this a TS override?

Reply:
I'll try to remember this weird thing. We may want to make it better at one point (but not in this PR). Just wanted to know if you found out something about his. 🙂